config/materials/ISO1629_SBR_SHORE_A70.mtl

```diff
@@ -0,0 +1,79 @@
+﻿ND_RelParSet_K01 = {
+
+Name = RUBBER
+
+PARAMETERS =
+{
+  Name = PTC_MATERIAL_DESCRIPTION
+  Type = String
+  Default = 'SBR rubber. ISO1629 Shore A 70.'
+  Access = Full
+},
+{
+  Name = TEMPERATURE
+  Type = Real
+  Default = 0.000000e+00 C
+  Access = Full
+},
+{
+  Name = PTC_INITIAL_BEND_Y_FACTOR
+  Type = Real
+  Default = 5.000000e-01
+  Access = Full
+},
+{
+  Name = PTC_BEND_TABLE
+  Type = String
+  Default = ''
+  Access = Full
+},
+{
+  Name = PTC_MATERIAL_TYPE
+  Type = Integer
+  Default = 9
+  Access = Locked
+},
+{
+  Name = PTC_FAILURE_CRITERION_TYPE
+  Type = String
+  Default = 'NONE'
+  Access = Locked
+},
+{
+  Name = PTC_FATIGUE_TYPE
+  Type = String
+  Default = 'NONE'
+  Access = Locked
+},
+{
+  Name = PTC_MATERIAL_SUB_TYPE
+  Type = String
+  Default = 'LINEAR'
+  Access = Locked
+},
+{
+  Name = PTC_MASS_DENSITY
+  Type = Real
+  Default = 1.650000e+00 g/cm^3
+  Access = Full
+},
+{
+  Name = PTC_MATERIAL_MODEL_DEF_BY_TESTS
+  Type = Bool
+  Default = NO
+  Access = Full
+},
+{
+  Name = PTC_MATERIAL_MODEL
+  Type = String
+  Default = 'NEO-HOOKEAN'
+  Access = Locked
+},
+{
+  Name = PTC_TENSILE_ULTIMATE_STRESS
+  Type = Real
+  Default = 3.000000e+00 MPa
+  Access = Full
+}
+
+}
```

Comment:
ND_RelParSet_K01 = {

Name = ISO1629_SBR_SHORE_A70

PARAMETERS =
{
  Name = PTC_MATERIAL_TYPE
  Type = Integer
  Default = 9
  Access = Locked
},
{
  Name = PTC_FAILURE_CRITERION_TYPE
  Type = String
  Default = 'NONE'
  Access = Locked
},
{
  Name = PTC_FATIGUE_TYPE
  Type = String
  Default = 'NONE'
  Access = Locked
},
{
  Name = PTC_MATERIAL_SUB_TYPE
  Type = String
  Default = 'LINEAR'
  Access = Locked
},
{
  Name = PTC_MATERIAL_DESCRIPTION
  Type = String
  Default = 'SBR rubber. Shore A 70.
Wrong params in hyperelastic model & shtm props'
  Access = Full
},
{
  Name = TEMPERATURE
  Type = Real
  Default = 0.000000e+00 C
  Access = Full
},
{
  Name = PTC_BEND_TABLE
  Type = String
  Default = ''
  Access = Full
},
{
  Name = PTC_MASS_DENSITY
  Type = Real
  Default = 1.650000e-06 kg/mm^3
  Access = Full
},
{
  Name = PTC_TENSILE_ULTIMATE_STRESS
  Type = Real
  Default = 3.000000e+03 kPa
  Access = Full
},
{
  Name = PTC_INITIAL_BEND_Y_FACTOR
  Type = Real
  Default = 5.000000e-01
  Access = Full
}

}